cylc/flow/cfgspec/suite.py

```diff
@@ -351,27 +351,26 @@
             supports use of the 365 (never a leap year) and 366 (always a leap
             year) calendars.
         ''')
-        Conf('runahead limit', VDR.V_STRING, desc='''
+        Conf('runahead limit', VDR.V_STRING, 'P5', desc='''
             Runahead limiting prevents the fastest tasks in a suite from
             getting too far ahead of the slowest ones, as documented in
             :ref:`RunaheadLimit`.
 
-            This config item specifies a hard limit as a cycle interval
-            between the slowest and fastest tasks. See also
-            :cylc:conf:`[..]max active cycle points` which defines the limit
-            as a number of cyles.
-
-            Example: ``PT12H`` - for a 12 hour limit under ISO 8601 cycling.
-        ''')
-        Conf('max active cycle points', VDR.V_INTEGER, 3, desc='''
-            Runahead limiting prevents the fastest tasks in a suite from
-            getting too far ahead of the slowest ones, as documented in
-            :ref:`RunaheadLimit`.
-
-            It allows up to ``N`` (default 3)
-            consecutive cycle points to be active at any time, adjusted up if
-            necessary for any future triggering.
-        ''')
+            This limit on the number of consecutive cycle points is specified
+            by an interval between the least and most recent: either an integer
+            interval (e.g. ``P3`` -  works for both :term:`integer cycling` and
+            :term:`datetime cycling`), or a time interval (e.g. ``PT12H`` -
+            only works for datetime cycling). Alternatively, if a raw number is
+            given, it will be taken to mean that number of hours, though this
+            usage is deprecated.
```

Comment:
I don't think this format of `Pn` (e.g. `P3`) for a number limit is good, because if you have
```ini
[[graph]]
    P2 = foo  # where foo fails
```
That means the cycle points will be `foo.1`, `foo.3`, `foo.5` etc. But the runahead limit will stall at `foo.5` not `foo.3` because `foo.5` is actually the 3rd cycle point.

So `runahead limit = 3` instead of `P3` would make more sense. But the trouble is it would break the backwards compatibility where the raw number is interpreted as `PT3H`

Reply:
Well, actually, it wouldn't break backwards compatibility because this issue only occurs in integer cycling mode. It might make it a bit harder to parse the value though.

Reply:
The alternatives would be:

* Stick to `P<n>` but document the hell out of it.
* Come up with another prefix character (confusing).

Reply:
> Stick with `P<n>`, just document the hell out of it.

That seems the best option to me, if we can't afford to lose the raw hours back compat.

Reply:
Sadly the old feature was never actually deprecated and is widely used.